### DISCUSSION TITLE
Add a "Submit all" button for "Repairs"

### DISCUSSION THREAD

**Discussion by ildar170975** [home-assistant/frontend#18634] [Dashboard]
Sometimes (usually after upgrading to a new HA version) a see that:
- none of integrations is loaded (incl. "Sun");
- list of issues to be repaired becomes huge (which is empty usually).

To "repair" an issue - I need to go inside every issue and press a "Submit" button:

![image](https://github.com/home-assistant/frontend/assets/71872483/8f7d0286-4da0-4a2f-bbe4-41395f3ea82c)

Surely this is not a "Fix the issue" button, it just means "Yes, I know about the issue".

Please add a new button for "mass-submitting" all issues:

![image](https://github.com/home-assistant/frontend/assets/71872483/079adecf-afec-49e1-bfc6-a38b87f4dc28)

**Comment by frenck** [comment]
That is impossible. Not all repairs are done when hitting Submit. Some may trigger a completely new UI flow requiring you to make a decision.

**Reply by ildar170975** [reply]
Some issues only have a "submit" button.
For them it would be great to mass-select & "submit" w/o entering every issue.
